doc/POLICIES.md

```diff
@@ -0,0 +1,71 @@
+# Bundler Policies
+
+This document is an attempt to record the policies and processes that are used to govern the Bundler project—it’s not fixed or permanent, and will likely evolve as events warrant.
+
+### Our Goals
+
+0. Treat everyone like a valuable human being, worthy of respect and empathy. No exceptions.
+1. Strive to empower the developers who use Bundler. For example, there is no such thing as user error, only insufficient UX design.
+2. When it does not disempower users, strive to empower contributors. For example, potential contributors should be able to set up a complete development and testing environment with a single command.
```

Comment:
it -> Bundler ?

Reply:
maybe... "When the outcome will not disempower users, strive to empower contributors."?